tests/math_builtin_api/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ foreach(cat ${MATH_CAT})
     FILE_PREFIX "math_builtin_${cat}"
     EXT "cpp"
     INPUT "math_builtin.template"
-    EXTRA_ARGS -test ${cat} -marray true -fragment-size 150
+    EXTRA_ARGS -test ${cat} -marray true -fragment-size 230
```

Comment:
While you are here, you could factorized out this function or macro and add your PR comment as a code comment. :smile:

Reply:
@steffenlarsen, could you please reply to this request?

Reply:
So sorry for missing this! I think I need to look at my email filters.

@keryell, do you mean making the magic number here a CMake option with this default value?

Reply:
Yes for example, with a description in the option explaining what it does and how it sets some trade-off.